<launch>
<param name="use_sim_time" value="false" />

  <group ns="planner">
      <remap from="openni_points" to="/wav_pcl"/>
      <remap from="map" to="/map"/>
      <remap from="odom" to="/odometry/filtered"/>
      
      <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
      <remap from="/planner/cmd_vel" to="/cmd_vel"/>
      <remap from="/planner/motion_event" to="/motion_event"/>
      <!-- <remap from="/planner/move_base_simple/cmd_vel_ack" to="/cmd_vel_ack"/> -->
      <param name="controller_frequency" value="5.0" />

      <rosparam file="$(find wavnodes)/config_trj/costmap_common_params.yaml" command="load" ns="global_costmap" />
      <rosparam file="$(find wavnodes)/config_trj/costmap_common_params.yaml" command="load" ns="local_costmap" />
      <rosparam file="$(find wavnodes)/config_trj/local_costmap_params.yaml" command="load" />
      <rosparam file="$(find wavnodes)/config_trj/global_costmap_params.yaml" command="load" />
      <rosparam file="$(find wavnodes)/config_trj/base_local_planner_params.yaml" command="load" />
      </node>
   </group>

</launch>